app/api/PostReview.js

```diff
@@ -9,12 +9,50 @@ export default async (req, res) => {
   const queryParams = [reviewer, rating_one, rating_two, rating_three, text, course_number, quarter, professor];
   
   // Execute the query
-  pool.query(query, queryParams, (err, results) => {
+  await pool.query(query, queryParams, (err, results) => {
     if (err) {
       console.error('Error executing MySQL query:', err);
       res.status(500).json({ error: 'Internal Server Error' });
     } else {
       res.json({ success: true, message: 'Data inserted successfully' });
     }
   });
+
+  // SQL query to calculate the average ratings from all the reviews for the course
+  const avgQuery = `
+    SELECT 
```

Comment:
I think the query is this:

SELECT 
    AVG(Reviews.rating_one) AS avg_rating_one,
    AVG(Reviews.rating_two) AS avg_rating_two,
    AVG(Reviews.rating_three) AS avg_rating_three
FROM
    Reviews
JOIN
    Courses ON Reviews.class_id = Courses.class_id
WHERE
    Courses.number = ?

Also, you can just test this yourself and see if it updates the database to check. I wanted to check if it was right before merging, you should test it if u can (definitely test this query cuz I'm not 100% sure)